django_prices/forms.py

```diff
@@ -22,42 +22,18 @@ def __init__(
         *args,
         **kwargs
     ):
-        self.currency = currency
-        if isinstance(widget, type):
-            widget = widget(
-                currency=self.currency, attrs={"type": "number", "step": "any"}
-            )
-        super(MoneyField, self).__init__(*args, widget=widget, **kwargs)
+        self.default_currency = default_currency or ""
```

Comment:
`""` is not a valid currency code and the argument is not optional.

Reply:
You're obviously right, my fault.